src/controller/stationManagerController.js

```diff
@@ -0,0 +1,63 @@
+import StationManagerView from "../view/stationManagerView.js";
+import StationDAO from "../model/stationDAO.js";
+import { ERROR_CODE, ERROR_CODE_MSG } from "./stationManagerControllerError.js";
+
+export default class StationManagerController {
+  constructor() {
+    this.stationManagerView = new StationManagerView();
+    this.stationDAO = new StationDAO();
+    this.addEventAboutShowAllStations();
+  }
+  addEventAboutShowAllStations() {
+    document
+      .querySelector("#station-manager-button")
+      .addEventListener("click", () => this.showAllStations());
+  }
+  showAllStations() {
+    this.stationManagerView.makeHtml(this.stationDAO.getAllStations());
+    this.addEventAboutAddStation();
+    this.addEventAboutDeleteStation();
+  }
+  addEventAboutAddStation() {
+    document
+      .querySelector("#station-add-button")
+      .addEventListener("click", () => {
+        let stationName = document.querySelector("#station-name-input").value;
+        let errCode = this.isValidStationName(stationName);
+        if (errCode != ERROR_CODE.SUCCESS) {
```

Comment:
```suggestion
        if (errCode !== ERROR_CODE.SUCCESS) {
```


자바스크립트에서 == 는 값을 비교하고, ===는 타입을 기반으로 비교하기 때문에 엄격한 비교인 ===를 항상 쓰는게 조금 더 좋아보여서 리뷰했습니다.